.github/workflows/pull-requests.yaml

```diff
@@ -150,6 +151,15 @@ jobs:
       changed: ${{ needs.get-changed-modules.outputs.managers_changed }}
       artifact-id: ${{ needs.find-artifacts.outputs.workflow_for_artifact_download_id }}
 
+  pr-build-ivts:
+    name: Build the 'ivts' module
+    needs: [get-changed-modules, find-artifacts, pr-build-managers]
```

Comment:
this has to run after the OBR builds, as to build it the managers need to be available on a maven repo site to compile against ? or is the build of this just going to refer to artifacts in the build workspace?

Reply:
For PRs I have set the IVTs to run directly after the Managers have built as these builds just download artifacts in the build workspace so we don't care about the maven repo site being rebuilt/deployed.